packages/website/.eslintrc.js

```diff
@@ -17,7 +21,8 @@ module.exports = {
   rules: {
     'react/jsx-no-target-blank': 'off',
     'react/no-unescaped-entities': 'off',
-    'react-hooks/exhaustive-deps': 'error',
+    '@typescript-eslint/internal/prefer-ast-types-enum': 'off',
```

Comment:
we do not want to use enums in here, we do not import `typescript-eslint` on web